DigitalLearningSolutions.Web/ViewModels/LearningMenu/CourseCompletionViewModel.cs

```diff
@@ -17,6 +17,8 @@ public class CourseCompletionViewModel
         public int SectionCount { get; }

```

Comment:
`DiagnosticAttempts` can be private now I think? 🙂

Reply:
Yes, sorry about that! Removed in 08ae966 🙂